tests/Api/Campaigns/ContentsTest.php

```diff
@@ -14,11 +15,14 @@ class ContentsTest extends AbstractTestCase
 
 	public function testGetWorks(): void
 	{
+		/** @var Content $ret */
 		$ret = $this->getClient()->campaigns()->contents(self::CAMPAIGN_ID)->get();
 		$this->assertInstanceOf(Content::class, $ret);
 		$this->assertStringContainsString('<title>TestSubject 00</title>', $ret->archiveHtml);
 		$this->assertStringContainsString('<title>*|MC:SUBJECT|*</title>', $ret->html);
-		$this->assertNull($ret->plainText);
+		$rp = new ReflectionProperty(Content::class, 'plainText');
```

Comment:
Short answer too this is NO

I think the correct solution to this is to change ` Stefna\Mailchimp\Model\Campaign\Content`

from
```php
public string $plainText;
```
to
```php
public ?string $plainText = null;
```

And I think this is the solution for all places you get `must not be accessed before initialization` errors